### DISCUSSION TITLE
How to use rolling in a table with the dice roller in statblock?

### DISCUSSION THREAD

**Discussion by morinibh** [Obsidian-TTRPG-Community/fantasy-statblocks#446] [Q&A]
I would like to use rolling to determine an item in a table like in a dice roller:

|dice: 1d4|Item   |
|----------|-------|
|1-2         |Item1 |
|3-4         |item2 |
^table

`dice-mod: [[path#^table]]`

in the statblock:

```statblock
name_dice: `dice-mod: [[path#^table]]`
```

Is it possible?

**Comment by valentine195** [comment]
Plugin processors do not run inside code blocks, so this isn’t possible directly. You could possibly use templater and insert a template instead.
